### PR TITLE
Add Docker EULA for direct downloads

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add Docker EULA and DPA info on pages that contain direct download links.

### CODE REVIEW

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+By downloading Docker Desktop, you agree to the terms of the [Docker Software End User License Agreement](https://www.docker.com/legal/docker-software-end-user-license-agreement){: target="_blank" rel="noopener" class="_"} and the [Docker Data Processing Agreement](https://www.docker.com/legal/data-processing-agreement){: target="_blank" rel="noopener" class="_"}.
```
Comment (thaJeztah) [inline docker-for-mac/release-notes.md:14]: Should we create an include for this?

Comment (usha-mandya) [reply docker-for-mac/release-notes.md:14]: Good idea :) I'll create an include and update the files

Comment (usha-mandya) [reply docker-for-mac/release-notes.md]: Done. PTAL

**Review by StefanScherer — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**